week-1/interpret/percentage-change.js

```diff
@@ -1,22 +1,32 @@
 let carPrice = "10,000";
 let priceAfterOneYear = "8,543";
-
-carPrice = Number(carPrice.replaceAll(",", ""));
-priceAfterOneYear = Number(priceAfterOneYear.replaceAll(",", ""));
+carPrice = Number(carPrice.replace(/,/g, ""));
+priceAfterOneYear = Number(priceAfterOneYear.replace(/,/g, ""));
 
 const priceDifference = carPrice - priceAfterOneYear;
 const percentageChange = (priceDifference / carPrice) * 100;
 
 console.log(`The percentage change is ${percentageChange}`);
 
+
+
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
+// 2 functions are called in this code. `Number()` and `replaceAll()` each two times has have been called .
```

Comment:
There are _more_ than 2 function calls overall.
You've correctly identified `Number` and `replaceAll` are being called. But focus on one line like this:
`Number(carPrice.replace(/,/g, ""));` - how many function calls are there here?

Reply:
Number , replace , and i'm not sure the Regex inside the replace considered a function or not  `/,/g`